checkstyle-suppressions.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+    "http://checkstyle.sourceforge.net/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+  <suppress checks="AbbreviationAsWordInName"  files="LaunchDescriptor.java"/>
```

Comment:
Please do not suppress this check for the complete file. Suppress only the three lines where this is needed, for example using `@SuppressWarnings("checkstyle:AbbreviationAsWordInName")` after enabling `<module name="SuppressWarningsHolder" />` and `<module name="SuppressWarningsFilter" />`
as explained on https://checkstyle.sourceforge.io/config_filters.html#SuppressWarningsFilter
The rule shoult catch any names we may add in future.

Reply:
good idea, I'll try to get that to work